### PR TITLE
Provide API endpoints for update periodic task schedule

### PR DESCRIPTION
**Pull request by huiwangredhat**
Fixes [OLCS-618](https://issues.redhat.com/browse/OLCS-618)

### CODE REVIEW

File: openlcs/packages/views.py [hunk 1]
```diff
@@ -1264,3 +1270,161 @@ def get(self, request, *args, **kwargs):
...
+class CrontabScheduleViewSet(ModelViewSet):
+    """
+    API endpoint that allows CrontabSchedule to be viewed or edited
+    """
+    queryset = CrontabSchedule.objects.all()
+    serializer_class = CrontabScheduleSerializer
+
+    def list(self, request, *args, **kwargs):
+        """
+        List the exist crontab schedules.
```
Comment (hanchuntao) [inline openlcs/packages/views.py:1364]: `existing`

File: openlcs/openlcs/urls.py [hunk 2]
```diff
@@ -35,6 +35,10 @@
 router = HybridRouter()
 router.register(r'auth', auth_views.TokenViewSet, basename='auth')
 router.register(r'files', package_views.FileViewSet, basename='files')
+router.register(r'periodictask', package_views.PeriodicTaskViewSet,
+                basename='periodictask')
+router.register(r'crontabschedule', package_views.CrontabScheduleViewSet,
+                basename='crontabschedule')
```
Comment (hanchuntao) [inline openlcs/openlcs/urls.py:41]: It is better to order the urls.

File: openlcs/packages/views.py (hunk 1 above)
Comment (huiwangredhat) [reply openlcs/packages/views.py:1364]: Updated.

File: openlcs/openlcs/urls.py (hunk 2 above)
Comment (huiwangredhat) [reply openlcs/openlcs/urls.py]: Ordered.

**Comment by huiwangredhat** [comment]
Close [OLCS-618](https://issues.redhat.com/browse/OLCS-618)

**Pull request merged**